estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="estate_property_view_search" model="ir.ui.view">
+        <field name="name">estate.property.view.search</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <search string="Search Properties">
+                <field name="name"/>
+                <field name="postcode"/>
+                <field name="expected_price"/>
+                <field name="bedrooms"/>
+                <field name="living_area"/>
+                <field name="facades"/>
+                <field name="tags"/>
+
+                <filter string="Status" name="status" domain="['|', ('state', '=', 'new'), ('state', '=', 'offer_received')]"/>
```

Comment:
This filter name is strange, It doesn't really tell what it filters and i doubt you can guess it from what will result of the filtering. Maybe calling it "Pending" or something like that would make it clearer ?

```suggestion
                <filter string="Pending" name="pending" domain="['|', ('state', '=', 'new'), ('state', '=', 'offer_received')]"/>
```